.github/workflows/round_id_linker.yml

```diff
@@ -0,0 +1,12 @@
+name: "Round ID Linker"
```

Comment:
For this to work, we'd have to clone the round_linker repo and then replace the scrubby link regex in the index.js with a slimbus one first.